### PR TITLE
options: Don't crash when OpenSSL lacks SSLv3 support

### PR DESCRIPTION
**Pull request by ivan**
Fixes https://github.com/chfoo/wpull/issues/287

### CODE REVIEW

**Comment by ivan** [comment]
Specifically, python3.5 on Ubuntu 15.10 is compiled without SSLv3 support.

File: wpull/options.py [hunk 1]
```diff
@@ -897,14 +897,19 @@ def _add_http_args(self):
...
         self._ssl_version_map = {
-            'auto': ssl.PROTOCOL_SSLv23,
-            'SSLv3': ssl.PROTOCOL_SSLv3,
-            'TLSv1': ssl.PROTOCOL_TLSv1,
+            # PROTOCOL_SSLv23 also selects TLS protocols
+            'auto': ssl.PROTOCOL_SSLv23
         }
 
         if hasattr(ssl, 'PROTOCOL_SSLv2'):
             self._ssl_version_map['SSLv2'] = ssl.PROTOCOL_SSLv2,
```
Comment (chfoo) [inline wpull/options.py]: I'm not sure why I have trailing commas. They shouldn't be there and the new addition with the trailing commas seems to causes the related (ignore the phantomjs ones) tests to fail.

Comment (ivan) [inline wpull/options.py]: Oh, weird, I didn't even notice that.  Want me to update the PR?

Comment (chfoo) [inline wpull/options.py]: Ok, update.

**Comment by ivan** [comment]
Updated per code review.

**Pull request merged**